Sources/Apodini/Modifier/TypedRelationshipDestinationModifier.swift

```diff
@@ -0,0 +1,39 @@
+//
+// Created by Andreas Bauer on 20.01.21.
+//
+
+public struct TypedRelationshipDestinationModifier<H: Handler, To>: HandlerModifier {
```

Comment:
Even though it is not needed here I would suggest to constraint `To` to `ResponseTransformable`:
```suggestion
public struct TypedRelationshipDestinationModifier<H: Handler, To: ResponseTransformable>: HandlerModifier {
```
Every type that is returned by a `Handler` must conform to the protocol and therefore it might give a developer an additional hint if the developer passes in a type that can not be returned by a `Handler`.

Reply:
Not sure about that. As this would explicitly state that some type of Response<String> would be okay to pass. Even if this would be currently allowed, it is still semantically somewhat weird in my opinion.

Reply:
nit: Under what circumstances would we mark a relationship to an object that is not returned by a handler?
I think this is a fair constraint

Reply:
As discussed in the work meeting: We will leave it as is.

Reply:
As discussed in the meeting we will keep the `To` generic type unconstrained.